tests/databrowser.js

```diff
@@ -157,40 +156,13 @@ define([
                 });
             });
 
-            it("should add recursive ref on drag/drop", function(done) {
```

Comment:
I think recursive relationships should be possible to represent in the schema. The tree should only show three levels: case, parent, granparent. Still sorta think this test should not have been removed, but probably not a big deal.